Loop/Managers/SleepStore.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  SleepStore.swift
+//  Loop
+//
+//  Created by Anna Quinlan on 12/28/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+enum SleepStoreResult<T> {
+    case success(T)
+    case failure(Error)
+}
+
+enum SleepStoreError: Error {
+    case noMatchingBedtime
+    case unknownReturnConfiguration
+    case noSleepDataAvailable
+}
+
+extension SleepStoreError: LocalizedError {
+    public var localizedDescription: String {
```

Comment:
It doesn't look like we'll be surfacing these errors to the user, so you can remove the localized descriptions.